lib/parse.ts

```diff
@@ -11,25 +12,41 @@ import {
 	Track,
 	Waypoint,
 	Extensions,
+	Options,
 } from "./types"
 
+const DEFAULT_THRESHOLD = 0.000215 // m/ms - 0.215 m/s - 774000 km/h
+const DEFAULT_OPTIONS = {
+	removeEmptyFields: true,
+	avgSpeedThreshold: DEFAULT_THRESHOLD,
+}
+
 /**
  * Converts the given GPX XML to a JavaScript Object with the ability to convert to GeoJSON.
  *
  * @param gpxSource A string containing the source GPX XML
  * @param removeEmptyFields Whether or not to remove null or undefined fields from the output
  * @returns A ParsedGPX with all of the parsed data and a method to convert to GeoJSON
  */
-export const parseGPX = (gpxSource: string, removeEmptyFields: boolean = true) => {
+export const parseGPX = (
+	gpxSource: string,
+	removeEmptyFields: boolean = true
```

Comment:
Pass in options object to `parseGPX`, the same as with `parseGPXWithCustomParser`. Then merge this with the default options to fill in any unspecified values.